.github/workflows/dependency-submission.yml

```diff
@@ -20,3 +20,6 @@ jobs:
         java-version: 17
     - name: Generate and submit dependency graph
       uses: gradle/actions/dependency-submission@v3
+      env:
+        # Exclude all dependencies that originate solely in the following projects
+        DEPENDENCY_GRAPH_EXCLUDE_PROJECTS: 'workbench'
```

Comment:
not sure if the `:` is required at the beginning or not, but might be safest to include it

Reply:
updated